### ISSUE TITLE
Can I see the full error message in the outbox?

### ISSUE THREAD

**Issue by hughgrigg** [spatie/mailcoach-support#89]
I can see that there are errors when trying to send via SES in Mailcoach's Outbox, but as the error messages are truncated I cannot see what they are.

For example, I can see this:

```
Error executing "SendRawEmail" on "https://email.us-east-1.amazonaws.com"; AWS HTTP error: Client error: `POST https://email.us-east-1.amazonaws.com` resulted in a `400 Bad Request` response: <ErrorResponse xmlns="http://ses.amazonaws.com/doc/2010-12...
```

Is there any way to see the full error message so that I can try to diagnose what the problem is?

**Comment by hughgrigg** [comment]
I've also looked at these in the `mailcoach_sends.failure_reason` column, and they are also truncated there. Could they not be truncated?

**Comment by riasvdv** [comment]
I've fixed this in the new release we're working on, hopefully out soon!

**Comment by mabadir** [comment]
Hi @riasvdv, I am on v2.23.15, and I am having this problem. Is there something I need to do to get the full error message?

**Comment by riasvdv** [comment]
This is now fixed in 2.23.16 and 3.0.1, there was indeed still a limit on the message
